client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -51,66 +52,60 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
     >
       {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <label htmlFor="username">
```

Comment:
```suggestion
          <label htmlFor="name">
```

Ditto throughout!

Reply:
For consistency I've changed the text field and label to 'Username' and placeholder to 'Create a username' then username is disaggregated from first and last name which is more typical of login interfaces.